modules/upgrade.nix

```diff
@@ -11,5 +11,11 @@
       createDevShm = ''
         mkdir -p ${config.build.installationDir}/dev/shm
       '';
+      createProcStat = ''
```

Comment:
There already can be installs on 21.11 that don't have the file created, so the condition isn't fully accurate. But maybe we don't need to iron this out, see below.